### PR TITLE
feat(BpmnJsonConverter): add support for intermediate compensation throw event

### LINKED ISSUE #911: feat(BpmnJsonConverter): add support for intermediate compensation throw event
It seems that current version of BpmnJsonConverter doesn't support serialization and deserialization of intermediate compensation event. However the engine supports it.

### PR DESCRIPTION
**Pull request by alirezamirian**
closes #911

### CODE REVIEW

File: modules/activiti-json-converter/src/main/java/org/activiti/editor/language/json/converter/BaseBpmnJsonConverter.java [hunk 1]
```diff
@@ -16,39 +16,8 @@
...
-import org.activiti.bpmn.model.FlowNode;
-import org.activiti.bpmn.model.FormProperty;
-import org.activiti.bpmn.model.FormValue;
-import org.activiti.bpmn.model.Gateway;
-import org.activiti.bpmn.model.GraphicInfo;
-import org.activiti.bpmn.model.Lane;
-import org.activiti.bpmn.model.MessageEventDefinition;
-import org.activiti.bpmn.model.MessageFlow;
-import org.activiti.bpmn.model.MultiInstanceLoopCharacteristics;
+import org.activiti.bpmn.model.*;
```
Comment (vasiledirla) [inline modules/activiti-json-converter/src/main/java/org/activiti/editor/language/json/converter/BaseBpmnJsonConverter.java]: please don't use * import

**Comment by Salaboy** [comment]
@alirezamirian can you please rebase this to 6.x branch and or maybe develop? We want to make sure that we include this fix in the latest code if it still applies.

**Comment by alirezamirian** [comment]
Sure.
I will prepare a new PR in the following week.

**Comment by alirezamirian** [comment]
BTW, the localization extension element is not converted to/from json currently. I also fixed this in my local repo. I can create a PR for that too if you are interested.

**Comment by Salaboy** [comment]
@alirezamirian thanks we will appreciate that for sure. Feel free to join us in the gitter channel if you want to contribute with the project latest versions.

**Comment by alirezamirian** [comment]
Sorry for the delay. I just submitted [a new PR](https://github.com/Activiti/Activiti/pull/1594) targeting 
 **6.x** branch.

**Comment by alirezamirian** [comment]
And another one (#1597) for converting localization information (#1596).

**Comment by Salaboy** [comment]
Great! @alirezamirian thanks a lot! We will review and add comments.

**Pull request closed without merging**